README.md

```diff
@@ -65,6 +65,26 @@ to build a cohesive PaaS. It is not a turn-key PaaS solution.
 See the [getting started](http://paasta.readthedocs.io/en/latest/installation/getting_started.html)
 documentation for how to deploy PaaSTA.
 
+## Debugging PaaSTA
+
+You can debug PaaSTA by first running ``make my_life_easier``. This make target will add debugging settings in VS Code in launch.json file for running tests, the PaaSTA API, and the following PaaSTA CLI commands:
+
+ * logs
+ * validate
+ * status
+ * rollback
+ * mark-for-deployment
+
+ In order to run the debugger with the PaaSTA CLI commands, you can go to `Run and Debug` tab in VS Code and select the CLI command you want to run the debugger with, then press on `start debugging` button. The same follows for running PaaSTA tests except you would have to select `tox test`.
```

Comment:
Can we add some Images to this??

Reply:
Thanks, will add them

Reply:
imo, we should link to the vscode docs on how to use their debugger instead of including images that are likely to become out-of-date - if the image needs any sort of editing (e.g, adding arrows, circling buttons, etc), we're unlikely to come back here and regenerate them